pkg/kubelet/dockertools/container_gc.go

```diff
@@ -245,35 +281,53 @@ func (cgc *containerGC) GarbageCollect(gcPolicy kubecontainer.ContainerGCPolicy,
 	return nil
 }
 
-func (cgc *containerGC) removeContainer(id string, podNameWithNamespace string, containerName string) {
-	glog.V(4).Infof("Removing container %q name %q", id, containerName)
-	err := cgc.client.RemoveContainer(id, dockertypes.ContainerRemoveOptions{RemoveVolumes: true})
+func (cgc *containerGC) netContainerCleanup(containerInfo containerGCInfo) {
+	if containerInfo.isHostNetwork || !containerIsNetworked(containerInfo.containerName) {
```

Comment:
I'm assuming we need to check `!containerIsNetworked` because this is getting called in `deleteContainer` above?  Wondering if we could remove the need for this check within this method.  Shouldn't it really be an error to call `netContainerCleanup` on something that isn't a "net container"?